Cargo.toml

```diff
@@ -45,3 +45,8 @@ incremental = false
 codegen-units = 1
 inherits = "test"
 
+[package.metadata.cross.target.aarch64-unknown-linux-musl]
```

Comment:
Lets change `package.metadata.cross.target.aarch64-unknown-linux-musl` to `ockam_command.metadata.cross.target.aarch64-unknown-linux-musl`